upgrading/downgrade.adoc

```diff
@@ -13,45 +13,40 @@
 
 toc::[]
 
-== Overview
-
-Following an {product-title}
+After an {product-title}
 xref:../upgrading/index.adoc#install-config-upgrading-index[upgrade],
-it may be desirable in extreme cases to downgrade your cluster to a previous
-version. The following sections outline the required steps for each system in a
-cluster to perform such a downgrade for the {product-title} 3.10 to 3.9 downgrade
-path.
+you might need to downgrade your cluster to an earlier
+version. You can downgrade from {product-title} version 3.11 to version 3.10.
 
 [IMPORTANT]
 ====
-These steps are currently only supported for
+Downgrading a cluster to version 3.10 is supported for only
 xref:../install/index.adoc#planning-installation-types[RPM-based
-installations] of {product-title} and assumes downtime of the entire cluster.
+installations] of {product-title}, and you must take your entire cluster offline
+to downgrade.
 ====
 
 [[downgrade-verifying-backups]]
-== Verifying Backups
+== Verifying backups
 
-. The Ansible playbook used during the
-xref:../upgrading/index.adoc#install-config-upgrading-index[upgrade process]
-should have created a backup of the *_master-config.yaml_* file. Ensure this and
-the *_scheduler.json_* file exist on your masters:
+. Ensure that a backup of the *_master-config.yaml_* file,
+*_scheduler.json_* file, and the etcd data directory exist on your masters:
 +
 ----
 /etc/origin/master/master-config.yaml.<timestamp>
 /etc/origin/master/scheduler.json
+/var/lib/etcd/openshift-backup-xxxx
 ----
++ The Ansible playbook that you run during the
+xref:../upgrading/index.adoc#install-config-upgrading-index[upgrade process]
+creates these files. 
 
```

Comment:
According to "Preparing for an upgrade" in [in-place upgrades], above three files were backup manually. So here should be corresponding with there, and if need a reminder, just notice user that these files would be backup by ansible playbook during upgrade if user forgot to backup them.

Reply:
Would it be better to point out in the upgrading topic that these files are backed up by the playbook?

Reply:
> Would it be better to point out in the upgrading topic that these files are backed up by the playbook?

Yes. My suggestion is that not only saying our playbook will take a backup of these files, but also ensure user to backup them manually in case of any sudden(refer to https://bugzilla.redhat.com/show_bug.cgi?id=1613077#c3). So when it came to downgrade, they can use their manual backup as first choice, and if not, automatic backup is a last guarantee.

Reply:
LGTM now